src/components/grapher/Grapher.vue

```diff
@@ -68,17 +73,17 @@
           <!-- Bottom of the yard line numbers is approximately 11 steps from
               the sideline -->
           <text
-            v-for="(numberAndOffsetX, index) in yardLineNumberAndOffsetX"
-            :key="index + '-yardNum'"
+            v-for="numberAndOffsetX in yardLineNumberAndOffsetX"
+            :key="`${numberAndOffsetX}-yardNum`"
```

Comment:
Could this be simplified to 
```suggestion
            :key=`${numberAndOffsetX}-yardNum`
```